### ISSUE TITLE
[Bug] GPG Persistence 

### ISSUE THREAD

**Issue by shikhir-arora** [ForkIssues/Tracker#285]
GPG settings don't seem to persist; sometimes they work but if you view the settings after moving to a new repo it shows blank/default again. It's hard to know whether it was saved, so I just do it each time - but that gets annoying.

Would also like to mirror #271 -- having a "sign all commits" option would be great vs. per-repo. 🙂

**Comment by kervich** [comment]
Yes, it seems that it isn't. I would expect Fork to set/unset `commit.gpgSign` in repo config once a GPG key has been selected/cleared in the repo preferences UI.
